docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -513,6 +513,28 @@ This section might also come in handy when we'd like to avoid multiple expensive
 ----
 <1> `it` is the result of `item.callExpensiveLogicToGetTheValue(1,'foo',bazinga)`. The method is only invoked once even though the result may be used in multiple expressions.
 
+===== Let/Set Section
+
+This section is an alternative to `{#with}` which allows to specify multiple named parameters:
```

Comment:
We should still remove `#{with expr as binding}` because it does not have the semantics of `with`, and can be done with `#{let binding = expr}`.

Reply:
Ok. The semantics of `with` is not well-defined in Qute (or in templates in general) but I agree that it could be replaced with let/set.

Reply:
Well, `with` to modify the current scope is fine, it's the version with `as` which is problematic.

Reply:
Done.